### PR TITLE
Code formatter/#2359 create prettier config file

### PR DESCRIPTION
**Pull request by paslavskyi9roman**
Created a config file for Prettier

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 2,
+    "semi": true,
+    "singleQuote": true,
+    "editor.formatOnSave": true,
+    "bracketSpacing": true,
+    "printWidth": 200
+  }
```
Comment (JimHolman1199) [inline .prettierrc.json:9]: @paslavskyi9roman apply correct formating to this file

Comment (JimHolman1199) [inline .prettierrc.json:9]: @mr-pavlenko Do we need "arrow function brackets" set ro null here? 
or is printWidth - 200, how about 140 ? same we have in tslint file
what are the best practices here ?

Comment (JimHolman1199) [reply .prettierrc.json:9]: also do we need parser: 'typescript' here ?

Comment (paslavskyi9roman) [reply .prettierrc.json:9]: @JimHolman1199 Fixed with this commit 31744f2a10a6a01538bc0bf0a91e7b9bd94a37b1

Comment (paslavskyi9roman) [reply .prettierrc.json:10]: @JimHolman1199 Config has been changed with this commit 67a7bd2ba6989661ede73ec248c459e4646ec364

**Review by JimHolman1199 — APPROVED** [review]

**Review by yuriiLisunov — APPROVED** [review]

**Review by idzhersh — APPROVED** [review]

**Review by VitalyBorysenko — APPROVED** [review]

**Review by AlekOK — APPROVED** [review]

**Comment by idzhersh** [comment]
fix your conflicts. there are many

**Pull request closed without merging**